source/guide_part-db.rst

```diff
@@ -0,0 +1,161 @@
+
+.. highlight:: console
+.. author:: 927589452
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/partdb.png
+      :align: center
+
+#######
+Part-DB
+#######
+
+.. tag_list::
+
+Part-DB_ is a web-based database for managing electronic components and distributed under the GPL-2.0 License.
+
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`cronjobs <daemons-cron>`
+  * composer_
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.4:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.4'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Installation
+============
+
+``cd`` to your :manual:`document root <web-documentroot>`, then download the latest release of Part-DB and extract it:
+
+.. note:: The link to the latest version can be found at the `release page <https://github.com/Part-DB/Part-DB/releases`_.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
```

Comment:
There is a symlink in the homedir, so this can be much shorter.
```suggestion
 [isabell@stardust ~]$ cd html/
```

Reply:
I wasn't sure, if the symlink would get the same SE_Linux mode and basically copied this step from the nextcloud guide